.github/workflows/pass-fcrepo-acceptance-tests.yml

```
@@ -0,0 +1,40 @@
+name: pass-acceptance-testing
+
+# On demand only, run tests against Fedora based infrastructure
+on:
+  workflow_dispatch:
+
+env:
+  PI_FEDORA_EXTERNAL_BASE: "http://pass.local:8080/fcrepo/rest" # Used in the wait script
+
+jobs:
+  run_acceptance_tests:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Append hosts file
+      run: echo "127.0.0.1    pass.local" | sudo tee -a /etc/hosts
+
+    - name: Checkout pass-docker
+      uses: actions/checkout@v3
+
+    - name: Run pass-docker
+      run: |
+        docker-compose pull --quiet
+        docker-compose up -d
+
+    - name: Wait for startup
+      run: |
+        chmod +x ./wait-to-start.sh
+        ./wait-to-start.sh
+      shell: bash
+
+    - name: Checkout pass-acceptance-testing
+      uses: actions/checkout@v3
+      with:
+        repository: eclipse-pass/pass-acceptance-testing
```

I think we might want or even need to identify the branch that works for fedora based testing. If I recall correctly, I thought you made some mods to `main` that were demo specific?

```suggestion
        repository: eclipse-pass/pass-acceptance-testing
        ref: some-branch
```

---

Also something is up with the waiting as tests failed.